.github/ISSUE_TEMPLATE/release.md

```diff
@@ -1,86 +1,75 @@
 ---
-name: Publish a release
+name: Publish a major or minor release
 about: Publish a new Orchard Core release
 title: 'Release v'
 labels: release
 assignees: ''
 
 ---
-<!-- Be sure to also read https://docs.orchardcore.net/en/latest/topics/publishing-releases/. While the checklist is in a recommended order not every step depends strictly on the previous ones.  
-`<version name>` should be replaced with the current version, e.g. "1.8.0.". -->
 
-### Prepare the project
+# Release Major or Minor Preparation Guide
 
-Do some housekeeping on GitHub in the [main repo](https://github.com/OrchardCMS/OrchardCore).
+## Housekeeping on GitHub
 
-- [ ] Assign the milestone of the release's version to this issue.
-- [ ] Close remaining issues for the version (including merging corresponding pull requests if suitable) or assign them to the next one.
-- [ ] Assign all issues that were closed for an upcoming version (including a wildcard version like "1.0.x") to this version (milestone).
+- [ ] **Address Outstanding Issues**: Resolve any remaining issues for the current version.
+- [ ] **Merge Relevant Pull Requests**: Merge any applicable pull requests or assign outstanding issues to the next release.
 
-### Prepare the code and documentation
+## Code and Documentation Updates
 
-Update the source, so everything looks like on the new version.
+- [ ] **Create Release Branch**: Create a new release branch named `release/<version-name>` (e.g., `release/3.0.0`) from the `main` branch. This branch will serve as the foundation for upcoming patch releases and facilitate versioned updates.
```

Comment:
If the branch is for patch relases, then it should rather be named with the pattern `release/3.0.x`, because `3.0.0.` seems like it's exactly for that release. Or `release/3.x.x` for a new major version.

Reply:
updated